Dockerfile

```diff
@@ -25,5 +25,4 @@ COPY . $MVT_PATH
 WORKDIR $MVT_PATH
 RUN mv httpd.conf /usr/local/apache2/conf/httpd.conf
 
-RUN apt-get update
-RUN apt-get install -y ffmpeg wget python3 nano
+RUN apt-get update && apt-get install -y ffmpeg wget python3 nano
```

Comment:
Do we really need all in single line? if not then no changes required in this file